### PR TITLE
feat(ci): update base docker image and use the new image in ci

### PR DESCRIPTION
**Pull request by vdovhanych**
I have added dependencies for the node canvas as they were missing and switched the ci to this updated image. I would squash it into 1 commit for the merge.

### CODE REVIEW

**Review by mroz22 — APPROVED** [review]
that could help. but the mystery still remains why there were cases it did work... :thinking:

File: docker/ci-base/Dockerfile [hunk 1]
```diff
@@ -32,14 +32,19 @@ FROM ${CI_DOCKER_PROXY}node:14-buster
 
 RUN apt-get update && apt-get install -y \
     build-essential \
-    # required by cypress
+    # required by cypress and node
```
Comment (mroz22) [inline docker/ci-base/Dockerfile:35]: btw (out of the scope of this). we don't need dependencies for cypress in the base image as cypress is run in another container. this is some legacy leftover.

Comment (tsusanka) [reply docker/ci-base/Dockerfile:35]: It would be nice to remove the comment and the libs that are not needed!

**Review by tsusanka — APPROVED** [review]

**Comment by tsusanka** [comment]
Let's merge this after the release on Wednesday just to be sure.

**Comment by vdovhanych** [comment]
Hi @tsusanka , are we going to merge this or as we discussed the node version issue that could be related to a branch that has an older version on it and hasn't been worked on the issue disappeared? That in this case we probably don't need to change anything? This shouldn't break anything but just in case.

**Comment by tsusanka** [comment]
Let's merge this after July 14th just to be sure.

**Comment by vdovhanych** [comment]
Maybe we should close this as I'm not sure we need it anymore or do we still want to merge it? @tsusanka

**Comment by tsusanka** [comment]
Hm.. It is still a little bit of mystery that these dependencies were not added isn't it?

**Comment by vdovhanych** [comment]
> Hm.. It is still a little bit of mystery that these dependencies were not added isn't it?

Yeah, it is. The CI is now quite fine so who knows if this is really necessary but I would add it anyways as this is listed as dependency for node canvas.

**Comment by tsusanka** [comment]
Agreed. Merging then?

**Comment by vdovhanych** [comment]
Ping @tsusanka

**Pull request merged**